src/components/Recipes/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react'
+import { Link, useLocation } from 'react-router-dom'
+
+import { fetchRecipes } from 'api'
+import { IRecipe } from 'types'
```

Comment:
all types and api calls in single files? so it will be very small app, right?

Reply:
Yep, for now I'm not planning to make it big app.